src/02-js/01-ordenar/index.js

```diff
@@ -0,0 +1,58 @@
+// CASO 1
+/*const entrada = [456, 234, 78, 2, 8, 90, -12, 4, -23, 0, -100, 50];
+function ordenarMen(a,b){ return a-b; }
+console.log(entrada.sort(ordenarMen));
+
+function ordenarMay(a,b){ return b-a; }
+console.log(entrada.sort(ordenarMay));*/
+
+
+// CASO 2
+/*const entrada = ['asd', '123', '0a24', '1', '!', '_hola', '-6d', 'ASD', "??"];
+console.log(entrada.sort());
+console.log(entrada.reverse());*/
```

Comment:
@Camila1953, cambiar este caso para que se parezca a lo que sugerí para el primer caso, sobre todo en cuanto a que una única función debe ser capaz de ordenar de forma ascendente y descendente, el código en inglés y los console.log para facilitar la lectura por consola de los resultados de cada caso al ejecutar el script.

Tambien sugiero lo siguiente para tal vez con un input más amplio ver más facilmente si el resultado obtenido es correcto, me pase con carácteres especiales, era para que veas como los ordena y si te daba curiosidad investigar porque los ordena de esa forma.

```suggestion
// CASO 2
/*const entrada = ['asd', '123', '0a24', '1', '!', '_hola', '-6d', 'ASD', '??', 'algo', '?', 'algo más', 'papá', 'piso', 'papa', 'pelota'];
console.log(entrada.sort());
console.log(entrada.reverse());*/
```

Reply:
@Camila1953 veo que extendiste este array como lo sugerí cierro esta conversación